src/lang/en-US.ts

```diff
@@ -452,6 +468,37 @@ const CaptureCreate: ResolvedIntlConfig['messages'] = {
     'captureCreate.testNotification.desc': `Your capture succeeded in a dry run and can be saved.`,
 };
 
+const CaptureEdit: ResolvedIntlConfig['messages'] = {
+    'captureEdit.heading': `${RouteTitles['routeTitle.captureEdit']}`,
+    'captureEdit.details.heading': `Capture Details`,
+    'captureEdit.ctas.materialize': `Materialize Collections`,
+    'captureEdit.instructions': `The name and destination of your existing capture.`,
+    'captureEdit.missingConnectors': `No connectors are installed. You must install a source connector to edit a capture.`,
+    'captureEdit.noAccessGrants': `You do not have the necessary ${CommonMessages['terms.permissions']} to edit this capture. Please contact an administrator.`,
+    'captureEdit.tenant.label': `Prefix`,
+    'captureEdit.config.source.doclink': `Connector Help`,
+    'captureEdit.config.source.homepage': `Home`,
+    'captureEdit.save.failed': `Capture edit failed. See below for details:`,
+    'captureEdit.editor.default': `Before you can edit the capture specification, you must fill out the Connection Configuration section and click "${CTAs['cta.generateCatalog.capture']}." `,
+    'captureEdit.finalReview.instructions': `The following catalog was generated from the details you provided. To make changes, edit the YAML file directly. Click "${CTAs['cta.saveEntity']}" to proceed.`,
```

Comment:
"The following specification was generated..."

Reply:
Updated.